### PR TITLE
added rss feed for discover page

### PR DESCRIPTION
**Pull request by coleGillespie**
#106 feed can be found at host/rss

### CODE REVIEW

File: lib/posts.js [hunk 1]
```diff
@@ -243,6 +244,51 @@ var setPagination = function (defaultKey, request, next) {
...
+    }
+
+    rs.pipe(concat(function (posts) {
+      posts.map(function (post) {
+        var p = post.value;
+
+        feed.item({
+            title: p.created,
+            description: p.content,
+            url: 'http://' + domain + port + '/post/post!' + p.postid,
```
Comment (ednapiranha) [inline lib/posts.js]: protocol is https though... and some might run servers that are http only.. so gotta change it to be dynamic

File: lib/posts.js [hunk 2]
```diff
@@ -243,6 +244,51 @@ var setPagination = function (defaultKey, request, next) {
   });
 };
 
+exports.getRss = function(request, reply){
```
Comment (ednapiranha) [inline lib/posts.js]: whitespace!

File: lib/posts.js [hunk 3]
```diff
@@ -243,6 +244,51 @@ var setPagination = function (defaultKey, request, next) {
...
+  setPagination('post!', request, function (err, streamOpt) {
+    if (err) {
+      return reply(Boom.wrap(err, 400));
+    }
+
+    var rs = db.createReadStream(streamOpt.stream);
+    var port = conf.get('port');
+    var domain = conf.get('domain');
+
+    if (conf.get('port') !== 80){
```
Comment (ednapiranha) [inline lib/posts.js]: whitespace

File: lib/posts.js [hunk 4]
```diff
@@ -243,6 +244,51 @@ var setPagination = function (defaultKey, request, next) {
...
+            description: p.content,
+            url: 'http://' + domain + port + '/post/post!' + p.postid,
+            author: p.name,
+            date: p.created
+        });
+
+      });
+
+      return reply(feed.xml()).header('Content-type', 'application/xml');
+
```
Comment (ednapiranha) [inline lib/posts.js]: remove extra line break

File: lib/posts.js (hunk 1 above)
Comment (coleGillespie) [inline lib/posts.js]: do we have any examples of this already happening in the app anywhere @ednapiranha ?

Comment (ednapiranha) [inline lib/posts.js]: use server.info.protocol

**Comment by coleGillespie** [comment]
@ednapiranha fixed

**Comment by ednapiranha** [comment]
oops, can you rebase?

**Comment by coleGillespie** [comment]
@ednapiranha ok, rebase complete. let me know if you need some more updates from me for merge

**Pull request merged**